src/stylesheets/core/mixins/_sidenav.scss

```diff
@@ -13,65 +13,67 @@
 
   a {
     border: none;
-    color: $color-base;
+    color: color('base-dark');
     display: block;
-    font-family: $font-sans;
-    line-height: 1.3;
-    padding: 0.85rem 1rem 0.85rem $site-margins-mobile;
+    font-family: font-family('body');
+    line-height: line-height($theme-font-family-base, 3);
+    padding: units(1) units($theme-site-margins-mobile);
     text-decoration: none;
 
     &:hover {
-      background-color: $color-gray-lightest;
-      color: $color-primary;
+      background-color: color('base-lightest');
+      color: color('primary');
       text-decoration: none;
     }
 
     &:focus {
       outline-offset: 0;
       position: relative;
-      z-index: 1;
+      z-index: z-index(100);
     }
 
     &.usa-current { /* stylelint-disable-line selector-no-qualifying-type */
```

Comment:
Does this need the `a` specificity to work properly?

Reply:
I'm honestly not sure what this is for. What comes to mind is maybe the mobile nav? Removed in 90c4912 and hopefully everything still works as expected.

Reply:
Woops, thought this was about the z-index above, well the z-index is still removed now 😂.

Reply:
Probably for the best! :)

Reply:
Removed in e5047d1.